dxcherry/src/info/errors/customError.ts

```diff
@@ -0,0 +1,24 @@
+import { window } from 'vscode';
+import log from '../log';
+
+class CustomError extends Error {
```

Comment:
why class instead of two exported functions?

Reply:
because we can **throw** it now